admin/includes/functions/general.php

```diff
@@ -2666,6 +2666,17 @@ function zen_get_configuration_key_value($lookup) {
     return $lookup_value;
   }
 
+  function zen_get_configuration_group_value($lookup) {
+    global $db;
+    $configuration_query= $db->Execute("select configuration_group_title from " . TABLE_CONFIGURATION_GROUP . " where configuration_group_id ='" . zen_db_input($lookup) . "'");
+    $lookup_value= $configuration_query->fields['configuration_group_title'];
+    if ( $configuration_query->RecordCount() == 0 ) {
+      $lookup_value='<span class="lookupAttention">' . $lookup . '</span>';
```

Comment:
I think it's better to just return `$lookup` in this case.

But I'd rewrite the function like this, using an early-return:

```php
  function zen_get_configuration_group_value($lookup) {
    global $db;
    $configuration_query= $db->Execute("select configuration_group_title from " . TABLE_CONFIGURATION_GROUP . " where configuration_group_id ='" . (int)$lookup . "'");
    if ($configuration_query->RecordCount() == 0) {
      return (int)$lookup;
    }
    return $configuration_query->fields['configuration_group_title'];
  }
```